src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -372,7 +374,7 @@ bool RPCHandlerImpl::ConvertMessageToSO(
 
           smart_objects::SmartObjectSPtr msg_to_send =
               std::make_shared<smart_objects::SmartObject>(output);
-          v1_shema.attachSchema(*msg_to_send, false);
```

Comment:
@IGapchuk what is `v1_shema`? If you replaced it with class member then it most likely become unused now. Shouldn't you remove it entirely?

Reply:
@AKalinich-Luxoft Fixed in 942e36845e9eca9cf6390731be80a8d155e7ecf9.